spec/support/shared_examples/data_page_shared_example.rb

```diff
@@ -0,0 +1,7 @@
+shared_examples 'a data page' do |tables|
+  it 'contains all requires tables' do
+    tables.each do |table_name|
+      expect page.has_table? table_name
```

Comment:
So on the false positive thing, it turns out this snippet fails (says it returns false)

```ruby
expect(page.has_table?('DataTables_Table_0')).to be(true)
```

but this snippet passes

```ruby
expect page.has_table?('DataTables_Table_0')
```